### ISSUE TITLE
Input with igxMask and required attribute are marked as invalid initially

### ISSUE THREAD

**Issue by kdinev** [IgniteUI/igniteui-angular#7183]
## Description  
I have a required input with a mask. Out of my whole form, only this input is red initially, before the form is touched.

 * igniteui-angular version: 9.0.8
 * browser: all

## Steps to reproduce  

1. Define a required input with a mask
```html

      <igx-input-group class="order-form-input" type="border">
        <igx-prefix>
          <igx-icon title="Телефон">phone</igx-icon>
        </igx-prefix>
        <igx-prefix>
          +359
        </igx-prefix>
        <input id="orderPhone"
               #orderPhone
               class="input-phone-number"
               name="orderPhone"
               required
               igxInput
               [igxMask]="'000 00 00 00'"
               placeholder="--- -- -- --"
               [(ngModel)]="order.phoneNumber" />
        <igx-suffix *ngIf="orderPhone.value" (click)="order.phoneNumber = null">
          <igx-icon>clear</igx-icon>
        </igx-suffix>
      </igx-input-group>
```

## Result  
The input is red at load time.

## Expected result  
The input should behave like the rest of the required inputs that don't have masks.

**Comment by damyanpetev** [comment]
I believe this might be already resolved after the mask refactor in master (9.1.x). As far as I can tell this is the first part of https://github.com/IgniteUI/igniteui-angular/issues/6783

Should be a fairly simple patch to apply to the old state though (like delete one line)

**Comment by Lipata** [comment]
Fixed for versions 9.1.0 and above
